index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Frederick's-Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+       <nav class="nav-bar">
+        <h1 class="Logo">My Logo</h1>
+        <img src="Images/Union.png" alt="Three-Lines">
+       </nav>
```

Comment:
- [ ] Please the toolbar section should have the same margin-bottom right as  in the Figma design
![image](https://user-images.githubusercontent.com/74744787/176787448-e68f2d86-ef69-4f59-873d-373066ff9395.png)